rest/snapshot_test.go

```diff
@@ -764,3 +817,156 @@ var assetSnapshotsTestData = []string{
 		"type": "stocks"
 	}`,
 }
+
+var optionsSnapshotsTestData = []string{
+	`{
+		"name": "AAPL $50.00 call",
+		"market_status": "open",
+		"ticker": "O:AAPL230512C00050000",
+		"type": "options",
+		"last_quote": {
+		  "ask": 123.1,
+		  "ask_size": 90,
+		  "bid": 122.95,
+		  "bid_size": 90,
+		  "last_updated": 1683731850932649728,
+		  "midpoint": 123.025,
+		  "timeframe": "REAL-TIME"
+		},
+		"last_trade": {},
+		"session": {},
+		"break_even_price": 173.025,
+		"details": {
+		  "contract_type": "call",
+		  "exercise_style": "american",
+		  "expiration_date": "2023-05-12",
+		  "shares_per_contract": 100,
+		  "strike_price": 50
+		},
+		"greeks": {},
+		"underlying_asset": {
+		  "change_to_break_even": -0.11,
+		  "last_updated": 1683732072879546553,
+		  "price": 173.135,
+		  "ticker": "AAPL",
+		  "timeframe": "REAL-TIME"
+		},
+		"error": "",
```

Comment:
you can leave these properties off for the successful responses. they'll only be there for error responses